scripts/qa_crowbarsetup.sh

```diff
@@ -4781,18 +4782,26 @@ function onadmin_crowbar_nodeupgrade
         fi
 
         if [[ $want_nodesupgrade ]]; then
-            # FIXME: crowbarctl command can time out easily
-            # Do not use it until https://bugzilla.novell.com/show_bug.cgi?id=997293 is fixed
-            # safely crowbarctl upgrade services
-            curl -s -X POST -H "$crowbar_api_v2_header" $crowbar_api/api/upgrade/services
+            safely crowbarctl upgrade services
 
-            if grep -q "failed" /var/lib/crowbar/upgrade/6-to-7-progress.yml ; then
+            wait_for 300 5 "grep current_step $upgrade_progress_file | grep -v services" "services step to finish"
+
+            if grep -q "failed" $upgrade_progress_file ; then
                 crowbarctl upgrade status
-                complain 12 "Services step has failed. Check the upgrade status."
+                complain 12 "'Services' step has failed. Check the upgrade status."
             fi
 
-            # safely crowbarctl upgrade nodes
-            curl -s -X POST -H "$crowbar_api_v2_header" $crowbar_api/api/upgrade/nodes
+            # This is just a fake command now for OpenStack DB backup
+            # FIXME: replace with crowbarctl upgrade once available
+            curl -s -X POST -H "$crowbar_api_v2_header" $crowbar_api/api/upgrade/openstackbackup
```

Comment:
it's available now, and should be in the product by now. see https://github.com/crowbar/crowbar-client/releases/tag/v3.1.6